cookbook/iwoole_rgbw_table_lamp.rst

```diff
@@ -0,0 +1,174 @@
+IWOOLE Table Lamp
+===================================
+
+.. seo::
+    :description: Instructions for flashing and configuring IWOOLE Table Lamps.
+
+The IWOOLE Table Lamp is a RGBW lamp normally configured using the Tuya Smart App.
+It is available from various retailers online or directly from `iwoole.com <https://www.iwoole.com/product/category/wifi-smart-table-lamp/>`__.
+
+.. figure:: images/iwoole_rgbw_table_lamp.png
+    :align: center
+    :width: 50.0%
+
+1. Device overview
+==================
+.. note::
+
+    The following information relates to the desk model depicted above. A free-standing version and a desk version with a longer arm are available. They are likely to share the same internals and configuration, but I can not be certain. 
+
+My device had no external markings at all.
+
+The device is a basic 4-channel PWM RGBW light in a simple elegant housing.
+The LEDs are not individually addressable.
+No other sensors, outputs or status LEDs are available.
+
+The MOSFETs for the different color channels are connected as follows:
+
+- GPIO04: White
+- GPIO12: Green
+- GPIO13: Blue
+- GPIO14: Red
+
+1.1 Internal markings
+----------------------
+.. figure:: images/iwoole_rgbw_table_lamp_internal_1.jpg
+    :align: center
+    :width: 40.0%
+
+.. figure:: images/iwoole_rgbw_table_lamp_internal_2.jpg
+    :align: center
+    :width: 40.0%
+
+.. figure:: images/iwoole_rgbw_table_lamp_internal_3.jpg
+    :align: center
+    :width: 40.0%
+
+2. ESPHome configuration
+========================
+Since there is only one RGBW light to configure the .yaml file is fairly straightforward.
+Alternatively, you could configure each channel as a separate light if desired.
+I prefer to use the :ref:`color-interlock <rgbw_color_interlock>` option along with the configuration below. 
+
+2.1 Example configuration
+-------------------------
+.. code-block:: yaml
+
+    esphome:
+      name: "IWOOLE Table Lamp"
+      #ESP type is ESP8266EX with 1MB flash
+      platform: ESP8266
+      board: esp01_1m
+
+
+    # Standard configuration
+    wifi:
+      ssid: "yourwifinetwork"
+      password: "iamverysecure"
+      ap:
+        ssid: "IWOOLE Table Lamp Fallback Hotspot"
+        password: "safefallbackpassword"
+    captive_portal:
+    logger:
+    api:
+      password: "ialwaysforgetthisone"
+    ota:
+      password: "enteryourownpasswordhere"
+
+
```

Comment:
I think this would be the only thing I would change/remove as most people will be starting from scratch and know what they are doing, or using the wizard which generates this part.

```suggestion
```

Reply:
Yeah, I can see how that would remove some clutter. Thanks for the suggestion.